api/app/tests/__init__.py

```diff
@@ -1,12 +1,27 @@
-""" Util & common files
+""" Util & common files for tests
 """
-from typing import IO, Any
+from typing import IO, Any, Callable
+import os
 import datetime
 import json
 import importlib
 from app.db.models.common import TZTimeStamp
 
 
+def _load_json_file(module_path: str, filename: str) -> dict:
+    """ Load json file given a module path and a filename """
+    dirname = os.path.dirname(os.path.realpath(module_path))
+    with open(os.path.join(dirname, filename)) as file_pointer:
+        return json.load(file_pointer)
+
+
+def load_json_file(module_path: str) -> Callable[[str], dict]:
```

Comment:
The name of this method seems misleading. It's a function that returns a function right?

Reply:
Any suggestions? If I call it load_json_file_function it remains confusing. I think it's clear with the return type that it's returning a function - but I agree it's a confusing function.

Reply:
@Kyubinhan I don't know if renaming it has really made it more readable... have a look.

Reply:
Sorry did you change the name? Anyway yeah let's just stick to the old one. Naming is hard

Reply:
Yeah - I've reverted - adding curry to the mix just confuses things.